### DISCUSSION TITLE
Cronjob to update feeds

### DISCUSSION THREAD

**Discussion by robotfx** [FreshRSS/FreshRSS#3730] [General]
I've been trying the cronjobs below to auto-update feeds hourly, but neither works. I'm on Raspberry Pi 4, any idea how to make this?
```
1 * * * * wget -O - https://site.com/stiri/?c=feed&a=actualize >/dev/null 2>&1
1 * * * * php /var/www/site.com/html/p/?c=feed&a=actualize

```

**Comment by Frenzie** [comment]
The first might make sense, the second definitely does not. See https://freshrss.github.io/FreshRSS/en/admins/08_FeedUpdates.html for fully worked out examples.
